packages/clerk-js/src/core/auth/cookies/devBrowser.ts

```diff
@@ -0,0 +1,37 @@
+import { createCookieHandler } from '@clerk/shared/cookie';
+import { addYears } from '@clerk/shared/date';
+import { DEV_BROWSER_JWT_KEY } from '@clerk/shared/devBrowser';
+
+import { inCrossOriginIframe } from '../../../utils';
+
+export type DevBrowserCookieHandler = {
+  set: (jwt: string) => void;
+  get: () => string | undefined;
+  remove: () => void;
+};
```

Comment:
🙃 Nit picky, I'd prefer we had a common `CookieHandler` interface for these I think

Reply:
All 3 CookieHandlers do not have any conflict. I have no objection to merge them and use 1 CookieHandler for all. I think the current approach was used to use different parameters names and types for each of those handlers (eg clientUat has `set(client: ClientResource | undefined)` ) and to keep only the used operation for each handler (eg session handler does not have `remove()` method).
@nikosdouvlis do you think we should merge them?